beacon-chain/db/filesystem/blob.go

```diff
@@ -91,6 +91,13 @@ func (bs *BlobStorage) Save(sidecar blocks.VerifiedROBlob) error {
 		return err
 	}
 	partPath := fname.partPath()
+
+	// Ensure the partial file is deleted.
+	defer func() {
+		// Ignore return value, it's expected to fail if the save is successful.
+		_ = bs.fs.Remove(partPath)
```

Comment:
Static analysis won't let you ignore the error.

```
beacon-chain/db/filesystem/blob.go:98:3: Unhandled error for function call (github.com/spf13/afero.Fs).Remove (errcheck)
```

Consider logging a message here.